openapi/java.xml

```diff
@@ -37,18 +38,16 @@
                 <artifactId>client-java</artifactId>
                 <artifactVersion>1.0-SNAPSHOT</artifactVersion>
                 <sourceFolder>src/main/java</sourceFolder>
-                <localVariablePrefix></localVariablePrefix>
                 <serializableModel>false</serializableModel>
                 <bigDecimalAsString>false</bigDecimalAsString>
                 <fullJavaUtil>false</fullJavaUtil>
                 <hideGenerationTimestamp>false</hideGenerationTimestamp>
                 <dateLibrary>joda</dateLibrary>
                 <useRxJava>false</useRxJava>
                 <library>okhttp-gson</library>
-                <type-mappings>intstr.IntOrString=IntOrString,resource.Quantity=Quantity</type-mappings>
-                <import-mappings>
-                  IntOrString=io.kubernetes.client.custom.IntOrString,Quantity=io.kubernetes.client.custom.Quantity,V1Patch=io.kubernetes.client.custom.V1Patch
-                </import-mappings>
+                <typeMappings>int-or-string=IntOrString,quantity=Quantity</typeMappings>
```

Comment:
@wing328 for help

Reply:
How can I help? Can you share more details with me?